FEXCore/include/FEXCore/IR/IREmitter.h

```diff
@@ -332,6 +334,10 @@ friend class FEXCore::IR::PassManager;
       return Ptr;
     }
 
+    virtual void SaveNZCV() {
```

Comment:
oop, this should be override at the end instead of virtual. like in OpcodeDispatcher.h